backend/.prettierrc.json

```diff
@@ -0,0 +1,6 @@
+{
+  "semi": false,
+  "singleQuote": false,
```

Comment:
I put double quote since all the files have double quotes but you can change it if you want

Reply:
I guess, we should allow both single and double.